interfaces/apparmor/template.go

```diff
@@ -362,3 +362,35 @@ var defaultTemplate = []byte(`
 ###SNIPPETS###
 }
 `)
+
+// classicTemplate contains apparmor template used for snaps with classic confinement.
```

Comment:
Please also add this:

```
The classic template intentionally provides no confinement and is used simply to ensure
/// that processes have the proper command-specific security label instead of 'unconfined'.